locales/en-US/server.ftl

```diff
@@ -1,21 +1,16 @@
-// Localization for Server-side strings of Firefox Screenshots
-//
-// Please don't localize Firefox, Firefox Screenshots, or Screenshots
-
-
-
-// Global phrases shared across pages, prefixed with 'g'
-[[ global ]]
+### Localization for Server-side strings of Firefox Screenshots
+### Please don't localize Firefox, Firefox Screenshots, or Screenshots
```

Comment:
Should we add Terms for these?

Reply:
That would invalidate a ton of existing translations. So, I wouldn't be happy about it.

We might think about introducing terms and using them in new strings for the future.